platform/windows/os_windows.cpp

```diff
@@ -719,7 +719,7 @@ LRESULT OS_Windows::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
 						pressrc = 0;
 					}
 				}
-			} else if (mouse_mode != MOUSE_MODE_CAPTURED) {
```

Comment:
For the reference, I checked and this code has been this way since Godot was open sourced in 2014, so noone to blame for it :)